### PR TITLE
The avian revolution v0.3 - add more emotes & refactor existing emote code & sounds

### LINKED ISSUE #2044: *whistle isn't an emote.
Nuff said.

### PR DESCRIPTION
**Pull request by atakiya**
## About The Pull Request

Adds more emotes (some with sound!)
Refactors existing emotes.
Sound balancing and editing across the board to reduce unintended noise imbalance.
Also adds a tiny bit of flavourtext to some emotes for AI and cyborgs.

Added: *bawk, *caw, *caw2, *whistle
Ported: *blep, *bork, *hoot, *growl, *woof, *baa, *baa2, *wurble, *awoo2, *rattle, *cackle
Ported - for silicons: *dwoop, *yes, *no

## Why It's Good For The Game
Streamlined code is always good (if done properly)
More variety for people's choices on emotes.

## Changelog
:cl:
add: Added NEW emotes: *bawk, *caw, *caw2, *whistle
add: Added MORE EMOTES, ported from virgo: *blep, *bork, *hoot, *growl, *woof, *baa, *baa2, *wurble, *awoo2, *rattle, *cackle
soundadd: Added emote sounds: bawk.ogg, caw.ogg, caw2.ogg
soundadd: Ported: baa.ogg, baa2.ogg, bork.ogg, growl.ogg, hoot.ogg, long_awoo.ogg, rattle.ogg, woof.ogg, wurble.ogg, dwoop.ogg, cackle_yeen.ogg
soundadd: Ported for silicons: *dwoop, *yes, *no
refactor: Denoised and mixed modular emote soundfiles to mono, balanced sound volume
refactor: Refactored emote code a bit
/:cl:

### CODE REVIEW

**Comment by Sethnov** [comment]
am going to love this.

**Comment by atakiya** [comment]
If you got any suggestions for emotes to add as well, do tell before I'm done with this :)
________________________________

**Comment by KathrinBailey** [comment]
Coo? Maybe a squawk of utter panic when you, the bird, ends up getting caught by me?

**Comment by atakiya** [comment]
> 
> 
> Coo? Maybe a squawk of utter panic when you, the bird, ends up getting caught by me?

Coo could be done I think, pigeon ones are probably expected so I can find a good sound for that.
Squawk I tried to find a good sound before but.. I am really not sure which one to use.

**Comment by KasparoVy** [comment]
Fantastic. Been sorely missing some of these emotes
Would love to have the Hyena cackle in too from Virgo if at all possible

GJ on the sound balancing

**Comment by atakiya** [comment]
> 
> 
> Fantastic. Been sorely missing some of these emotes
> Would love to have the Hyena cackle in too from Virgo if at all possible
> 
> GJ on the sound balancing

Thank you. Still going over the files a bit so it's not final 
And consider the yeen cackle to be added.

File: modular_skyrat/modules/emotes/code/emotes.dm [hunk 1]
```diff
@@ -488,3 +488,11 @@
 	muzzle_ignore = TRUE
 	vary = TRUE
 	sound = 'modular_skyrat/modules/emotes/sound/voice/rattle.ogg'
+
+/datum/emote/living/cackle
```
Comment (KasparoVy) [inline modular_skyrat/modules/emotes/code/emotes.dm:495]: Omfg yes. Thank you Sir

**Comment by KasparoVy** [comment]
Loving that emotes are datumized here!

**Comment by atakiya** [comment]
IT IS DONE _**AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA**_

Kinda went overboard with this PR tbh, should've let it at a few additions and do the rest in another PR but HERE WE ARE AND I AM SURE AS HELL NOT GONNA DO THIS AGAIN.
~~at this point its more emote revolution than avian, but i guess thats what the 0.3 is for~~

**Comment by Gandalf2k15** [comment]
Phase one checks complete, will finalise tomorrow.

**Review by Gandalf2k15 — APPROVED** [review]
This looks fine. I'll merge it soon. Let's hope this works!

**Comment by atakiya** [comment]
> 
> Let's hope this works!

inb4
image

**Comment by KasparoVy** [comment]
Works on mine too *Shrug*

**Pull request merged**